data/sql/updates/db_world/0000_00_00_00.sql

```diff
@@ -0,0 +1,452 @@
+-- Cleanup for the total mess that's in the db
```

Comment:
this is not how we should include DB files, please use this:

https://www.azerothcore.org/wiki/Dealing-with-SQL-files

so you don't have to worry about the SQL file name and date, the BOT will take care of it later

Reply:
i know, it's placeholder till it's tested, I will generate it with the bash later